Kiosk/Bid Fulfillment/RegistrationPasswordViewController.swift

```diff
@@ -4,12 +4,16 @@ import Moya
 import Action
 
 class RegistrationPasswordViewController: UIViewController, RegistrationSubController {
+    
     @IBOutlet var passwordTextField: TextField!
     @IBOutlet var confirmButton: ActionButton!
     @IBOutlet var subtitleLabel: UILabel!
     @IBOutlet var forgotPasswordButton: UIButton!
+
     let finished = PublishSubject<Void>()
 
+    var provider: Networking!
```

Comment:
How do you feel about the idea of documenting `var` any time it's used? Looking at how `RegistrationPasswordViewController` is instantiated from a storyboard, it appears that this is a `var` just because of the need for late-initialization, but not because it's truly going to vary over the lifetime of the object. I think that's a legit use. Perhaps we can just annotate it with a `// for late initialization, not mutation`.